src/cli.rs

```diff
@@ -32,6 +32,14 @@ pub(crate) struct Cli {
     #[arg(short = 'l', long = "list")]
     pub list: bool,
 
+    /// Edit custom patch with `EDITOR`
+    #[arg(long, requires = "command", conflicts_with = "edit_page")]
+    pub edit_patch: bool,
+
+    /// Edit custom page with `EDITOR`
+    #[arg(long, requires = "command")]
+    pub edit_page: bool,
+
```

Comment:
I think I would switch the ordering of these two, because we also display patches after pages - it's always page & patch